code/game/objects/items/weapons/legcuffs.dm

```diff
@@ -93,16 +98,18 @@
 		to_chat(user, "<span class='notice'>You remove the signaler from [src].</span>")
 	return TRUE
 
+/obj/item/restraints/legcuffs/beartrap/proc/spring_trap(mob/user)
+	armed = FALSE
+	update_icon()
+	playsound(loc, 'sound/effects/snap.ogg', 50, TRUE)
+	if(!silent_arming)
+		user.visible_message("<span class='danger'>[user] triggers [src].", "<span class='userdanger'>You trigger [src].")
+
```

Comment:
Could you move this code to below where it's used?

Reply:
Resolved in [ce20a72](https://github.com/ParadiseSS13/Paradise/pull/21941/commits/ce20a72729e1183110f0a9ebde2fd060fde8b16c)